packages/api/src/Database/seeders/entities.ts

```diff
@@ -0,0 +1,12 @@
+import faker from 'faker';
+
+import { UserService } from '../../User/User.service';
+
+export const seedUsers = async (userService: UserService) => userService.create({
+  firstName: faker.name.firstName(),
+  lastName: faker.name.lastName(),
+  email: 'test@test.com',
+  password: 'test123'
+});
```

Comment:
Will every seeded User have the same email and password?
I imagine this is login credentials which should be unique, if so then it might be worth randomizing these values slightly to make them unique for each User. 

@tristanMatthias what are your thoughts on that?

Reply:
Yes, we should use faker.js for this (and all entities)

Reply:
I did it for testing purposes, I think it's annoying to always have different emails and passwords to log in and having the check it every time you seed the db, thoughts on this? And later on we can use the role if we have roles.

`
export const seedUsers = async (userService: UserService) => Promise.all(Array(5).fill(undefined).map(async (_, i) => {
  const firstName = faker.name.firstName();
  const lastName = faker.name.lastName();

  const user = await userService.create({
    firstName,
    lastName,
    email: `user${i}@test.com`,
    password: `user${i}123`
  });
  return user;
}));
`

<img width="790" alt="Screenshot 2020-04-25 at 07 31 04" src="https://user-images.githubusercontent.com/46815065/80258516-c1b91400-86c6-11ea-8482-00d9f64d0c83.png">